spec/message.abnf

```diff
@@ -0,0 +1,57 @@
+message = [s] *(declaration [s]) body [s]
+
+declaration = let s variable [s] "=" [s] "{" [s] expression [s] "}"
+body = pattern
+     / (selectors 1*([s] variant))
+
+pattern = "{" *(text / placeholder) "}"
+selectors = match 1*([s] selector)
+selector = "{" [s] expression [s] "}"
+variant = when 1*(s key) [s] pattern
+key = nmtoken / literal / "*"
```

Comment:
```suggestion
pattern   = "{" *(text / placeholder) "}"
selectors = match 1*([s] selector)
selector  = "{" [s] expression [s] "}"
variant   = when 1*(s key) [s] pattern
key       = nmtoken / literal / "*"
```

Reply:
Thanks for suggesting this. As I said above, I'd prefer to not column-align the ABNF for now because I don't think the names of production are final and I expect the next few weeks to bring a few changes.